lists/nhl/warmingUp/NhlPenalties.js

```diff
@@ -0,0 +1,72 @@
+var NhlPenalities = [
+{
+  "league": "nhl",
+  "type": "trivia",
+  "bucket": "sport",
+  "category": "NhlPenalities",
+  "keywords": "nhl penalties hockey",
+  "description": "Name All NHL Penalties",
+  "teams": [
+  {
+    "rank":"1",
+    "name": "High Sticking",
+    "stat": "Post",
```

Comment:
what is "post"? Since it's the same for every item, why is it needed in the individual object layer?